### PR TITLE
Feature/console log warn

### PR DESCRIPTION
**Pull request by eiditakahashi**
- Now checking for `console.log` on _web_ and _nodejs_ projects
- Updated `README`, removed `branch: "master"` #36

### CODE REVIEW

**Review by felipesabino — APPROVED** [review]

File: Dangerfile [hunk 1]
```diff
@@ -56,6 +57,7 @@ def checkForFileWeb(file)
   checkForEnginesVersion(file)
   checkForNpmInstallGlobal(file)
   validateSwiftFiles(file)
```
Comment (felipesabino) [inline Dangerfile]: I guess `validateSwiftFiles` can be removed from `web`, can't it?

**Review by felipesabino — APPROVED** [review]

**Comment by felipesabino** [comment]
@eiditakahashi tinha q atualizar o README :S

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ gem "danger", "4.2.1"
 1. Create a `Dangerfile` with
 ```ruby
 @platform = "nodejs" # Possible platforms are "nodejs", "ios", "android" and "web"
-danger.import_dangerfile(github: "indigotech/danger", branch: "master")
+danger.import_dangerfile(github: "indigotech/danger")
```
Comment (eiditakahashi) [inline README.md:18]: @felipesabino não era isso?

Comment (felipesabino) [reply README.md:18]: > Now checking for console.log on web and nodejs projects

sobre isso ☝️

**Pull request merged**